templates/advantage/index.html

```diff
@@ -336,7 +336,18 @@ <h4 class="p-card__title u-sv1 u-no-margin--bottom">
           {% endfor %}
         </div>
         <div class="col-12 u-align--right">
-          <a href="/advantage/subscribe{% if is_test_backend %}?test_backend=true{% endif %}" class="p-button--positive u-align--right" >Add new subscription</a>
+          <a 
+            href="/advantage/subscribe{% if is_test_backend %}?test_backend=true{% endif %}" 
+            class="p-button--positive u-align--right"   
+            onclick="dataLayer.push({
+              'event' : 'GAEvent',
+              'eventCategory' : 'Advantage',
+              'eventAction' : 'add-new-subscription',
+              'eventLabel' : 'Add new subscription',
```

Comment:
This would be confusing from GA as it seems to indicate that event is them purchasing a subscription but its just an action to show subscriptions.